.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
-snowsql_rt.*
\ No newline at end of file
+snowsql_rt.*
+transformation/bq/.DS_Store
```

Comment:
We were not including .DS_Store in the .gitignore as they are mac specific and easy revertible.
Anyway you can keep "DS_Store" that should cover all the cases, and remove "transformation/bq/.DS_Store"